bricksrc/equipment.py

```diff
@@ -6,32 +6,61 @@
 """
 equipment_subclasses = {
     "HVAC": {
-        OWL.equivalentClass: BRICK["Heating_Ventilation_Air_Conditioning_System"],
-        "tags": [TAG.Heat, TAG.Ventilation, TAG.Air, TAG.Conditioning, TAG.System],
-    },
-    "Heating_Ventilation_Air_Conditioning_System": {
-        OWL.equivalentClass: BRICK["HVAC"],
-        "tags": [TAG.HVAC],
+        "tags": [TAG.HVAC]
     },
     "Weather": {"tags": [TAG.Weather]},
-    "Electrical_System": {
-        "tags": [TAG.Electrical, TAG.System],
```

Comment:
I think we should replace `Electrical System` with `Electrical Equipment`, or something similar to group the more specific equipment types